db/seeds.rb

```diff
@@ -10,14 +10,23 @@
 require 'csv'
 
 # Load up the list of all companies and their name, symbol, sector, industry
+# Old header "Symbol","Name","LastSale","MarketCap","IPOyear","Sector","industry","Summary Quote",
+# Column index mapping:
+#  0: Symbol 
+#  1: Name 
+#  2: LastSale 
+#  3: MarketCap 
+#  4: IPOyear 
+#  5: Sector 
+#  6: industry 
+#  7: Summary Quote 
 CSV.foreach('db/csv/companies.csv',
-            headers: true,
-            header_converters: :symbol) do |row|
+            headers: true) do |row|
```

Comment:
According to http://ruby-doc.org/stdlib-1.9.2/libdoc/csv/rdoc/CSV.html#method-c-new we don't want `headers: true` because that treats the first row as a header, right?